### DISCUSSION TITLE
Permissions based Requests View

### DISCUSSION THREAD

**Discussion by nickelsh1ts** [sct/overseerr#806] [Feature Requests]
Currently, only admins and Request Managers can see active requests.
Can we implement either read-only of all requests for users to see what has already been requested, or if possible even better would be a set of permissions, users can only see pending, or active or all but not action etc.

See but not approve/deny.

**Comment by sct** [comment]
Sure! I wanted to make an option for this anyways. Can you open a feature request in Issues for this?

**Reply by nickelsh1ts** [reply]
Sure can! Cheers.
